### PR TITLE
[ENG-29950] feat: add billing endpoint

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

- Added fields for Billing GraphQL API
- Added mentions of the new endpoint in other GQL docs
- Added menu link to Billing GraphQL API Fields

[ENG-29950]: https://aziontech.atlassian.net/browse/ENG-29950?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/devtools/api-graphql/features/billing-fields.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+See each available field and their descriptions below.
+
+---
+
+## balanceFinancialEntry (Financial Entries)
+
+| Field | Description |
+| ----- | ----------- |
+| clientId | Client unique identifier on Azion. Example: `8437r` |
```
Comment (LucasGeneroso) [inline src/content/docs/en/pages/devtools/api-graphql/features/billing-fields.mdx:24]: Gostei da ideia de usar um Client ID seu 🧙🏽

**Review by LucasGeneroso — APPROVED** [review]
Ótimo trabalho! 🚀 🎖️

**Review by lfsigreja — APPROVED** [review]
menu lgtm

File: src/content/docs/en/pages/devtools/api-graphql/features/billing-fields.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+| entryType | Financial entry type. Example: `debit`, `trial_credit` |
+| description | Description to the balance entry that'll be disposed to the client bill and invoice. Example: `` |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/billing-fields.mdx:26]:
```suggestion
| description | Description of the balance entry that'll be disposed to the client bill and invoice |
```

File: src/content/docs/en/pages/devtools/api-graphql/features/features.mdx [hunk 3]
```diff
@@ -61,6 +61,19 @@ Four datasets are deprecated and were replaced by other datsets:
 
 <Button href="/en/documentation/devtools/graphql-api/features/gql-real-time-events-fields/" text="see Real-Time Events GraphQL API fields descriptions" variant="secondary"></Button>
 
+### Billing GraphQL API
+
+:::note
+The Billing GraphQL API is only available for accounts of the *Online Sales* type.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/features.mdx:67]:
```suggestion
The Billing GraphQL API is only available for *Online Sales* accounts.
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/billing-campos.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+---
+
+## balanceFinancialEntry (Entradas Financeiras)
+
+| Campo | Descrição |
+| ----- | --------- |
+| clientId | Identificador único do cliente na Azion. Exemplo: `8437r` |
+| entryType | Tipo de entrada financeira. Exemplo: `debit`, `trial_credit` |
+| description | Descrição da entrada de saldo que será lançada na fatura e na fatura do cliente. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/billing-campos.mdx:26]:
```suggestion
| description | Descrição da entrada de saldo que será lançada na fatura e na fatura do cliente |
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/devtools/api-graphql/recursos/billing-campos.mdx:26]: nota e boleto? fiquei na dúvida de qual seria a melhor trad aqui!

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**